@navikt/ds-react/accordion/Accordion.tsx

```diff
@@ -77,4 +102,35 @@ const Accordion = forwardRef<HTMLDivElement, AccordionProps>(
   }
 );
 
+Accordion.propTypes = {
```

Comment:
Er det verdt det å forvalte dobbelt sett med typer? (interface + propTypes) Hva gir det oss, og hvor stor synes vi kostnaden er?

Reply:
Er litt diverse meninger om kostnaden er verdt det, men fant noen ressurser om det: 
https://www.digitale-kumpel.ruhr/blog/react-typescript-and-prop-types
https://stackoverflow.com/questions/41746028/proptypes-in-a-typescript-react-application

Tldr: 
For brukere som bruker ren JS eller api calls så vil proptypes gjøre det enklere å finne bugs å feil når de bruker komponenten
For oss, så vil ikke react-docgen (proptable generering) fungere for de nye komponentene. Eks layout vil ikke returnere noen props på grunn av på grunn av `Layout as LayoutWithSubComponents`. Prop-table vil også gjøre at react-docgen genererer "bedre" data.

Reply:
Så jeg synes det er verdt å forvalte PropTypes

Reply:
Kan det være en grunn å gå vekk fra dot-notasjon? (med forbehold om at jeg misforsto hva du mente med `Layout as LayoutWithSubComponents`)

Reply:
Ja, hvis vi ønsker å ta i bruk dot-notasjon så må vi også bruke prop-types. Så vil spare mye ekstra kode og tid hvis vi dropper dot-notasjon da. Er kanskje litt for det

Reply:
God nok grunn for meg. Vi hadde jo ikke så mange tunge argumenter for eller mot sist.

Reply:
👍 Da går vi for komponenter uten dot-notasjon. Tar å fjerner prop-types fra denne prn, men beholder oppdateringene i komponent-interfacene